spec/unit/forge/forge_spec.rb

```diff
@@ -20,7 +20,10 @@
   # 3-byte ᚠ - http://www.fileformat.info/info/unicode/char/16A0/index.htm - 0xE1 0x9A 0xA0 / 225 154 160
   # 4-byte ܎ - http://www.fileformat.info/info/unicode/char/2070E/index.htm - 0xF0 0xA0 0x9C 0x8E / 240 160 156 142
   let (:mixed_utf8_query_param) { "foo + A\u06FF\u16A0\u{2070E}" } # Aۿᚠ
-  let (:mixed_utf8_query_param_encoded) { "foo%20%2B%20A%DB%BF%E1%9A%A0%F0%A0%9C%8E"}
+
+  # literal plus characters in the URI are converted to spaces prior to encoding
+  let (:mixed_utf8_query_param_encoded) { "foo%20%20%20A%DB%BF%E1%9A%A0%F0%A0%9C%8E"}
```

Comment:
On this... I think the intent was to show off encoding a literal `+`, so I think we need to change something here accordingly. (i.e. `mixed_utf8_query_param` matches up with `mixed_utf8_query_param_encoded`)